Submitted by:            Daniel Massey <daniel@thesonicmaster.net>
Date:                    2021-11-04
Initial Package Version: 3.0.3
Upstream Status:         Unknown
Origin:                  Self
Descrption:              Allow Python bindings to build with Python 3.10+.

--- old/libraries/libapparmor/m4/ac_python_devel.m4
+++ new/libraries/libapparmor/m4/ac_python_devel.m4
@@ -66,21 +66,6 @@ variable to configure. See ``configure -
         fi
 
         #
-        # Check if you have distutils, else fail
-        #
-        AC_MSG_CHECKING([for the distutils Python package])
-        ac_distutils_result=`$PYTHON -c "import distutils" 2>&1`
-        if test -z "$ac_distutils_result"; then
-                AC_MSG_RESULT([yes])
-        else
-                AC_MSG_RESULT([no])
-                AC_MSG_ERROR([cannot import Python module "distutils".
-Please check your Python installation. The error was:
-$ac_distutils_result])
-                PYTHON_VERSION=""
-        fi
-
-        #
         # Check for Python include path
         #
         AC_MSG_CHECKING([for Python include path])
